Sources/GRPCNIOTransportHTTP2TransportServices/HTTP2ClientTransport+TransportServices.swift

```diff
@@ -327,6 +368,54 @@ extension NWProtocolTLS.Options {
         `protocol`
       )
     }
+
+    switch tlsConfig.trustRoots.wrapped {
+    case .certificates(let certificates):
+      let verifyQueue = DispatchQueue(label: "certificateVerificationQueue")
```

Comment:
nit: should be prefixed with `io.grpc`, we can also drop "queue" from the name, it's a bit redundant here.

```suggestion
      let verifyQueue = DispatchQueue(label: "io.grpc.CertificateVerification")
```